src/components/layout/Header.tsx

```diff
@@ -80,10 +84,20 @@ const Header = ({ siteTitle, location }: Props) => {
                     ))}
                 </div>
 
-                <div className="hidden self-center md:inline-flex">
+                <div className='flex items-center space-x-2 relative top-1'>
+                  <button
+                    onClick={toggleSidebarPosition}
+                    className="hidden md:inline-flex items-center justify-center w-8 h-8 bg-primary text-white rounded hover:text-opacity-75"
+                  >
+                    <ArrowsRightLeftIcon className='h-4 w-4' />
+                  </button>
+
```

Comment:
I don't think the button to switch sides should be here on the navbar because not all pages have the sidebar and it looks weird for the about page and the FAQ page to have such button.

Can you move it to some place inside the sidebar? For example, near the download button or on the bottom of the sidebar.